Arrowgene.Ddon.GameServer/Characters/EquipManager.cs

```diff
@@ -168,5 +168,19 @@ public void HandleChangeEquipList(DdonGameServer server, GameClient client, Char
                 }
             }
         }
+        public void GetEquipTypeandSlot(Equipment equipment, string uid, out EquipType equipType, out byte equipSlot)
```

Comment:
Should we return a tuple `(EquipType, byte)` instead of using out parameters?

Reply:
It's fine either way, out params are more C# like, but a tuple works too and is probably more intuitive for people not used to it.